source/source_test.go

```diff
@@ -14,6 +14,7 @@
 
 package source
 
+/*
```

Comment:
Why commenting out the entire source_test? I would like to see some tests for the source.

Looks like `destination` didn't have tests either before.